spec/classes/puppet_server_passenger_spec.rb

```diff
@@ -6,10 +6,15 @@
     next if exclude_test_os() and exclude_test_os.include?(os)
     next if os_facts[:osfamily] == 'windows'
     context "on #{os}" do
+      let(:pre_condition) do
+        'include ::puppet'
+      end
```

Comment:
init.pp and params.pp are not evaluated, therefore e.g. passenger_pre_start is null and the tenary operator that sets http_pre_start is not evaluated correctly (because it only accepts true and false).